cli/src/cluster_query.rs

```diff
@@ -84,6 +84,13 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .takes_value(true)
                         .help("Stop after submitting count transactions"),
                 )
+                .arg(
+                    Arg::with_name("lamports")
+                        .long("lamports")
+                        .value_name("NUMBER")
+                        .takes_value(true)
+                        .help("Number of lamports to transfer for each transaction [default: 1]"),
```

Comment:
By adding `.default_value("1")` here, you can remove the explicit "[default: 1]"

Reply:
Heh, too hurrying me. I just couldn't notice the API usage around the code... 6fc7bcde24b5d42ac26cbf63d77288d8ac5de90b